app/views/users/show.html.erb

```diff
@@ -1 +1,37 @@
-<h1>This shows a particular user from the DB<h1>
+<%= stylesheet_link_tag "application"%>
+
+<div class="all_users"> 
+    <h1>A particular user</h1>
+    <a href=<%= "/users/#{@user.id}/posts/" %> >
+    <div class="user-container">
+
+        <img src=" <%= @user.photo %>" alt="User profile" width="130" height="130"><br>
+       
+            <div class="single-user">
+                <h2>User Name:  <%= @user.name %></h2><br>
+                <div class="singler-user-p">
+                     <p >No.of Posts: <%= @user.posts_counter %></p><br>
+                </div>
+            </div>
+    </div>   
+    <a> 
+</div>
+<div class="particular_user">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+</div>
+<% @user.display_posts.each do |post| %>
+    <a href=<%= "/users/#{@user.id}/posts/#{post.id}" %> >
+        <div class="particular_user">
+            <h3>Post #<%= post.id %></h3><br>
+            <p><%= post.text %></p><br>
+            <div class="user-com-likes">
+                <p>Comments: <%= post.comments_counter %>, </p>
+                <p>Likes: <%= post.likes_counter %></p>
+            </div>
+        </div>    
+    <a>
+<% end %>
+<div class="user-btn">
+  <button type="button">See all posts</button>   
```

Comment:
- [ ] Kindly add a path for this button when the user click on this button should map to user posts path